src/lib/components/comparisonTable/hooks/useComparisonTable.ts

```diff
@@ -5,7 +5,7 @@ import { ArrowValues } from '../components/TableArrows';
 
 export const useComparisonTable = () => {
   const [showMore, setShowMore] = useState<boolean>(false);
-  const [headerWidth, setHeaderWidth] = useState(0);
+  const [headerWidth, setHeaderWidth] = useState(1400);
```

Comment:
Just to get some context here, how did you get to the value `1400`?

Reply:
1400 seemed like a good compromise value as to what a container can be, although it is a pretty artificial and random number.

In a way, it doesn't really matter, as long as it isn't `0` or a really small value since the width of cells will always show a minimum value (tablewidth is the headerWidth value here):
`max-width: min(var(--firstColumnWidth, 288px), calc(var(--tableWidth) / 2));`

However, the goal here is moving away from zero to avoid the width changes jump that are visible. It could even be 2000 or bigger.

Reply:
If I understand correctly, the flakiness comes from `headerWidth` starting with `0` and then getting resized inside the `useEffect` on line #26?

Reply:
Exactly, that is what It seems to be causing the flakiness, resizeObserver and useEffect take some time to apply the final width size.

Reply:
It seems like a good enough solution for me. The `headerWidth` will be set to the correct value once the component finishes mounting.
I'm just wondering if this will indeed prevent the flakiness. Is there a case when the `headerWidth` starts bigger than it should and then the `useEffect` squeeze it back to the correct size?

Reply:
From what I was testing it seems like that never happens because of this line setting the max-width of cells:
`max-width: min(var(--firstColumnWidth, 288px), calc(var(--tableWidth) / 2));`

This makes the max width be the minimum value of either `288px` or `tablewidth / 2` which only fires if table width is a really low value like zero.

The only issue is the scrolling header on mobile that I'm setting the max-width to be `calc(100vw - 32px)`.

Reply:
Just gave your fix a spin and it seems to not break anything that I was able to test, fingers crossed that this will indeed fix the flakiness 🤞 😄

Reply:
Same here, really hope this is the cause! 🤞🤞